tests/python_helics/test_message_federate.py

```diff
@@ -19,10 +19,10 @@ def mFed():
         pass
 
     # Create Federate Info object that describes the federate properties #
-    fedinfo = h.helicsFederateInfoCreate()
+    fedinfo = h.helicsCreateFederateInfo()
 
     # Set Federate name #
-    h.helicsFederateInfoSetFederateName(fedinfo, "TestA Federate")
+    h.helicsFederateInfoSetCoreName(fedinfo, "CoreA Federate")
```

Comment:
these are very different calls,  in one you are setting the name of the core to connect to and in the previous version you were setting the desired name of the federate.

Reply:
Yes. The name of the federate now does not seem to be set by a function anymore. It is set when creating the federate. 

```
HELICS_EXPORT helics_federate helicsCreateValueFederate (const char *fedName, const helics_federate_info_t fi, helics_error *err);
```

This takes the federate name.

Reply:
See this line [here](https://github.com/GMLC-TDC/HELICS-src/pull/501/files#diff-78391ea64970da9b0bb6eb7148788be9R43).